api/.air.toml

```diff
@@ -0,0 +1,32 @@
+root = "."
+tmp_dir = "tmp"
+
+[build]
+  cmd = "make build"
+  bin = "tmp/main"
+  full_bin = "tmp/main --mongo-database=core --mongo-username=root --mongo-password=example --mongo-hosts=localhost:27017 --redis-address=localhost:6379 --environment=Development --fresh=true --seed=true --tls-disable=true --hydra-admin-url=http://localhost:4445 --hydra-public-url=http://localhost:4444 --login-templates-directory=pkg/apps/login/templates --login-client-id=login --login-client-name=login --login-client-secret=somesecret --login-iam-host=localhost:9000 --login-iam-scheme=http --web-templates-directory=pkg/apps/webapp/templates --web-client-id=webapp --web-client-secret=webapp --web-client-name=webapp --web-iam-host=localhost:9000 --web-iam-scheme=http --web-cms-host=localhost:9000 --web-cms-scheme=http --listen-address=:9000 --base-url=http://localhost:9000"
```

Comment:
I encourge you to see if you can make this more readable.
1) are any of these flags redundant?
2) can you reuse any variables (localhost, port 9000 etc)
3) can you replace this entire command with an alias/command that's predefined somewhere else?